### DISCUSSION TITLE
How do I dismiss the action sheet when selecting an item?

### DISCUSSION THREAD

**Discussion by mheere** [Daniel-Ioannou/flutter_adaptive_action_sheet#24] [Q&A]
Hi -great looking lib but how do I dismiss the sheet when I select an Item?
Thanks

**Comment by mheere** [comment]
Ah - I found a way I think - in the onPressed action (after doing what you need to do) call:

`Navigator.of(context, rootNavigator: true).pop();`

instead of `Navigator.pop(context);`

This makes sense I suppose - in order to ensure that the popup is shown it is brought up through the rootNavigator so we also need to pop it like so!

**Reply by G-Gamja** [reply]
Thanks to your solution, I solved the problem what you met :)
